src/components/Nav/Nav.js

```diff
@@ -1,25 +1,66 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import { Link } from 'react-router-dom';
 import './Nav.scss';
 
-class Nav extends React.Component {
+class Nav extends PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      navActiveNumber: 0,
+      isNavTransper: false,
+    };
+  }
+
+  componentDidMount = () => {
+    window.addEventListener('scroll', this.handleScroll);
+  };
+
+  componentWillUnMount = () => {
+    window.removeEventListener('scroll', this.handleScroll);
+  };
+
+  handleScroll = () => {
+    if (window.pageYOffset > 0) {
+      this.setState({ isNavTransper: true });
+    } else if (window.pageYOffset === 0) {
+      this.setState({ isNavTransper: false });
+    }
+  };
+
+  navActiveHandler = index => {
+    if (index === 'object') {
+      this.setState({ navActiveNumber: 0 });
+    } else this.setState({ navActiveNumber: index + 1 });
```

Comment:
index==="object"로 판단하는건 옳지 않아보입니다!
index가 object타입이 될 수 있다는 것 자체가 이 함수를 봤을때 혼동이 있을 여지가 많아보입니다!
```suggestion
this.setState({navActiveNumber:index})

onClick={()=>this.navActiveHandler(0)}
onClick={()=>this.navActiveHandler(index+1)}
```

Reply:
흑........................................................